### ISSUE TITLE
Express middlewear doesn't seem to add the session

### ISSUE THREAD

**Issue by braceandbracket** [clerkinc/clerk-sdk-node#18]
Hey Team! I was wondering if there was something more than what was documented in the docs to have the `session` object append to the `req` object using `ClerkExpressWithSession`. Also when I use `ClerkExpressRequireSession,  I get a 404. I'm wondering if there is something I'm missing that's causing the session token to not validate. 

Here is my API call from the client (I'm using Axios `withCredentials: true`): 
```
GET /resources?_clerk_session_id=sess_1rnwWH... 
Accept: application/json, text/plain, */*
Cookie: __stripe_mid=67a5b8...; __cfduid=d44772...; __session=eyJhbG...;
```

in my express app I have:
```
...
app.use(ClerkExpressWithSession())
app.use(validateUser()) // custom middlewear
...
```

So when I try to access `req.session` it's undefined in my custom validateUser middlewear.

**Comment by yourtallness** [comment]
Hello @braceandbracket,

Responded to you on Intercom to get the issue figured out & will repost the solution here for anyone that might benefit from it.

**Comment by yourtallness** [comment]
Hello again @braceandbracket

We just released version 0.4.1 of the node SDK which supports better logging in order to ease debugging (https://www.npmjs.com/package/@clerk/clerk-sdk-node).

﻿If you set `CLERK_LOGGING=true` in your environment, the error messages from the server should be logged in the console.

Could you give it a try and check the console output when you perform a request to your express server?

**Comment by yourtallness** [comment]
@braceandbracket Closing for now.

Please don't hesitate to re-open if this issue occurs again.
